devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -531,6 +537,14 @@ private def handleAccessAlarmReport(cmd) {
 			map = [ name: "codeChanged", value: "0 set", descriptionText: "${getStatusForDescription('set')} \"$codeName\"", isStateChange: true ]
 			map.data = [ codeName: codeName, notify: true, notificationText: "${getStatusForDescription('set')} \"$codeName\" in $deviceName at ${location.name}" ]
 			break
+		case 24: //KeyWe manual unlock
+			map = [ name: "lock", value: "unlocked", data: [ method: "manual" ] ]
+			map.descriptionText = "Unlocked manually"
+			break
+		case 25: //Keywe manual lock
```

Comment:
These two values should be in hex (0x18 and 0x19) to match the Z-Wave documentation and surrounding code.